### ISSUE TITLE
How to configure this plugin?

### ISSUE THREAD

**Issue by csvke** [hongkongkiwi/node-xiaomi-mitv2-remote#1]
Is there more materials to read regarding the configuration of this plugin?
I have a Mi TV 4 65" and would like to control the tv over Home App in iOS and set scenes with it.
Thanks.

**Comment by hongkongkiwi** [comment]
I no longer have a Xiaomi tv :-( but I used a proxy to get the api calls from the Xiaomi app to create this.

**Comment by csvke** [comment]
Thanks for the reply.

I am absolutely new to the whole HomeBridge eco system...

Was glancing through the code you have written, it doesn't seem to be following the structure of the example plugins <https://github.com/nfarina/homebridge/tree/master/example-plugins>
I wonder your code was written long ago before HomeBridge introduced a more structural way of managing plugins?

For example, I reckon plugin folders should have a prefix of homebridge- 
<https://github.com/nfarina/homebridge#installing-plugins> but your's don't. I wonder if it's one of the reasons why the plugin is not registered.

Secondly, is your plugin written as an 'accessory'?

I read through your code, took some of the REST API GET and put it to Postman and tested. It works!
I cannot find anything in the internet regarding the MiTV Assistant Server API reference but I can confirm it still works in port 6095 and for example a GET of 
http://your.mitv.ip:6095/controller?action=keyevent&keycode=menu
would work with my MiTV 4 65"
so I think its worth pursuing to maybe rewrite or maintain the plugin.

It's also surprising there's no authentication, nor https encryption, was implemented with the MiTV Server Assistant API... guess it's another debate.

Any information or experience sharing is much appreciated. I am based in HK.

**Comment by csvke** [comment]
Btw since the REST API calls is working in plain url, I reckon there's no need to have a proxy and to use the Xiaomi app at all? My wild guess was that the proxy you setup (although I don't know how to) kind of reverse engineered part of the API's JSON / Resources / Argument schema...

**Comment by hongkongkiwi** [comment]
Hi there, sorry geeze I didn't even notice this question.

I used Charles which is a Mac OSX Man in the middle proxy. I installed the fake ssl certificate on my phone and then simply used charles to monitor the requests between app and tv, it's quite straightforward if you want to give it a try.

I don't actually have a TV anymore, so I'm not able to test :/ I didn't like the xiaomi tv advertisements ! so I got a samsung.
